packages/spacecat-shared-rum-api-client/src/functions/opportunities/high-organic-low-ctr.js

```diff
@@ -11,27 +11,32 @@
  */
 
 import trafficAcquisition from '../traffic-acquisition.js';
-import { getCTRByUrl, getSiteAvgCTR } from '../../common/aggregateFns.js';
+import { getCTRByUrlAndVendor, getSiteAvgCTR } from '../../common/aggregateFns.js';
 
 const DAILY_EARNED_THRESHOLD = 5000;
 const CTR_THRESHOLD_RATIO = 0.95;
 const DAILY_PAGEVIEW_THRESHOLD = 1000;
+const VENDORS_TO_CONSIDER = 5;
 
 function convertToOpportunity(traffic) {
   const {
-    url, total, ctr, paid, owned, earned, siteAvgCTR,
+    url, total, ctr, paid, owned, earned, vendors, siteAvgCTR, ctrByUrlAndVendor,
   } = traffic;
 
-  return {
+  const topVendors = Object.entries(vendors)
+    .sort((a, b) => b[1].total - a[1].total).slice(0, VENDORS_TO_CONSIDER);
+  const opportunity = {
     type: 'high-organic-low-ctr',
     page: url,
     screenshot: '',
     trackedPageKPIName: 'Click Through Rate',
     trackedPageKPIValue: ctr,
     pageViews: total,
     samples: total, // todo: get the actual number of samples
+    siteAverage: siteAvgCTR,
```

Comment:
this change violates the spec for the `ExperimentationOpportunity` schema: https://github.com/adobe/spacecat-api-service/blob/main/docs/openapi/schemas.yaml#L514

Reply:
@ekremney is it ok if I add this `siteAverageCTR` to `ExperimentationOpportunity` spec?